tests/unit/infrastructure/application/bracken/test_bracken_profile.py

```diff
@@ -29,156 +29,177 @@
 
 
 @pytest.mark.parametrize(
-    "columns",
+    "profile",
     [
-        (
-            "name",
-            "taxonomy_id",
-            "taxonomy_lvl",
-            "kraken_assigned_reads",
-            "added_reads",
-            "new_est_reads",
-            "fraction_total_reads",
+        pd.DataFrame(
+            OrderedDict(
+                [
+                    ("name", ["root"]),
+                    ("taxonomy_id", [1]),
+                    ("taxonomy_lvl", ["root"]),
+                    ("kraken_assigned_reads", [100]),
+                    ("added_reads", [0]),
+                    ("new_est_reads", [100]),
+                    ("fraction_total_reads", [1.0]),
+                ]
+            )
         ),
         pytest.param(
-            (
-                "name",
-                "taxonomy_id",
-                "kraken_assigned_reads",
-                "added_reads",
-                "new_est_reads",
-                "fraction_total_reads",
+            pd.DataFrame(
+                OrderedDict(
+                    [
+                        ("name", ["root"]),
+                        ("taxonomy_id", [1]),
+                        ("kraken_assigned_reads", [100]),
+                        ("added_reads", [0]),
+                        ("new_est_reads", [100]),
+                        ("fraction_total_reads", [1.0]),
+                    ]
+                )
             ),
             marks=pytest.mark.raises(
                 exception=SchemaError, message="column 'taxonomy_lvl' not in dataframe"
             ),
         ),
         pytest.param(
-            (
-                "name",
-                "taxonomy_id",
-                "taxonomy_lvl",
-                "kraken_assigned_reads",
-                "added_reads",
-                "new_est_reads",
-                "fraction_total_reads",
-                "foo",
+            pd.DataFrame(
+                OrderedDict(
+                    [
+                        ("name", ["root"]),
+                        ("taxonomy_id", [1]),
+                        ("taxonomy_lvl", ["root"]),
+                        ("kraken_assigned_reads", [100]),
+                        ("added_reads", [0]),
+                        ("new_est_reads", [100]),
+                        ("fraction_total_reads", [1.0]),
+                        ("foo", [1.0]),
```

Comment:
Do we still need to identify what that column is?

Reply:
What do you mean? This test just ensures that there will be an error for an unknown extra column "foo".

Reply:
Ah misunderstood that (I didn't see it in other files), all good!

Reply:
Yeah, there is a slight inconsistency in what exactly is tested per profiler.